### ISSUE TITLE
dl3007 false negative with certain comments

### ISSUE THREAD

**Issue by aogier** [hadolint/hadolint#303]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

If before `FROM` there is at least one comment line whose last non-whitespace char is "\", DL3007 fails.

eg.

```dockerfile
# multiple optional lines
# multiple optional lines
# \
# multiple optional lines
FROM debian:latest
```
version: Haskell Dockerfile Linter v1.16.0-0-g475ad73

Thank you, regards!

**Comment by lorenzo** [comment]
duplicate of #294 

A patch is already under review for this. Thanks for your report

**Comment by aogier** [comment]
this seems a general idiosyncrasy with backslashes rather than a duplicate of another unrelated (from a non-author standpoint) issue. Thank you for your work

**Comment by lorenzo** [comment]
@aogier I just pushed a branch with a fix for the issue, should be realised shortly

**Comment by aogier** [comment]
thank you for the update! Regards
